client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-renderer.scss

```diff
@@ -1,3 +1,11 @@
 .pattern-list-renderer__pattern-list-item {
 	padding: 0;
+
+	.components-popover__content {
+		white-space: normal;
+		width: auto;
+		max-width: 280px;
+		margin: 0 auto;
+	}
```

Comment:
These changes are to break the line because we have patterns with long names.

|BEFORE|AFTER|
|---|---|
|<img width="376" alt="Screenshot 2566-04-12 at 14 05 27" src="https://user-images.githubusercontent.com/1881481/231377216-d16abc1a-eb96-46d4-bf6d-d920920dcb86.png">|<img width="335" alt="Screenshot 2566-04-12 at 14 04 46" src="https://user-images.githubusercontent.com/1881481/231377311-023065a7-4c20-4059-a884-8bda72afe1f7.png">|


The rest of the changes are required to center the tooltip and limit the width. I tried with `100%` and other ways to not use a static width but they didn't work. `280px` works well for all tooltips in header, footer and sections.

Reply:
The issue is its position is related `pattern-list-panel__wrapper` (width: 348px). We can add `position: relative` to `pattern-selector__body` and then `max-width: 100%` will works as expected

Reply:
Done ✅